### DISCUSSION TITLE
Best way to animate a sequence of flood maps?

### DISCUSSION THREAD

**Discussion by rsignell-usgs** [TerriaJS/terriajs#6284] [Q&A]
We have a sequence of COG images from SAR showing the recent flooding on the Red River in the USA/Canada:

https://data.amerigeoss.org/ameriterria-disasters/#clean&proxy/_60s/https://raw.githubusercontent.com/EarthBigData/openSAR/master/code/terriajs/json/red_river.json

If you look at the catalog, you can see we are using `url-template` catalog item to access data from a tile service that is actually created on the fly, reading cloud-optimized data files directly from S3. 

Is there any way to animate a sequence of imagery like this over time?

**Comment by nf-s** [comment]
To animate time-series imagery you could to create a WMS layer with a time parameter.

At the moment, I believe we don't have the ability to animate a sequence of `url-template` catalog items across time.

**Reply by nf-s** [reply]
I have created an issue for this https://github.com/TerriaJS/terriajs/issues/6316
